src/test/resources/com/atlassian/performance/tools/report/api/result/real-results/DSR/7.13.1-J713SPT-JPTC-110 : Data Center : 1.3M attachments/virtual-users/virtual-user-node-1/test-results/418217d8-8c78-4cd4-b0ca-249c1c8dc326/action-metrics.jpt

```diff
@@ -0,0 +1,423 @@
+{"label":"Log In","result":"OK","duration":"PT2.642S","start":"2018-12-20T11:46:39.418Z","virtualUser":"418217d8-8c78-4cd4-b0ca-249c1c8dc326"}
```

Comment:
Files changed: 412
Lines added: 490,711

Yikes, the diff is too big. It's possible to ignore these parts of the diff via many tools, but it'd be even better if we didn't have to do that.
I'd zip it, like we did with https://github.com/atlassian/report/blob/60071cdc693c489ce919958dbd697945f176580e/src/test/resources/com/atlassian/performance/tools/report/api/result/real-results/QUICK-54/c5.18xlarge%2C%202%20nodes%2C%20run%201.zip

Reply:
That's a good point. Will do it.